### DISCUSSION TITLE
"blueprints" directory in the repository should be written with lower-case

### DISCUSSION THREAD

**Discussion by Kogaa5** [QualiTorque/qualitorque.github.io#39] [Documentation requests] [answered]
"blueprints" directory in the repository **must** be written with lower-case?
Do we want to add it to the documentation?

**Comment by Shnitselon** [comment, marked as answer]
@TomerAdmon 

resolved this request in https://github.com/QualiTorque/qualitorque.github.io/pull/59/commits/dfd1e532e81ecfe3b6bf925c3bbf700f233328b3

This discussion can be closed once we approve and merge pull request https://github.com/QualiTorque/qualitorque.github.io/pull/59
